site/layouts/_default/taxonomy.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+
+<html>
+    <head>
+        {{ partial "head.html" .}}
+    </head>
+<body>
+    {{ partial "nav2.html" .}}
+    <div class="container">
+        <div class="row">
+            <div class="col-md-9 doc-content">
+                <div class="well well-sm">
+                    <strong>Items in category <code>{{ .Title | lower }}</code></strong>
+                    <ul class="list-unstyled" style="margin-top: 20px">
+                        {{ range .Data.Pages }}
+                            {{ .Render "li" }}
+                        {{ end}}
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
Please add the footer here:
```suggestion
    </div>
    {{ partial "footer.html" .}}
```

Reply:
Ok.